### DISCUSSION TITLE
amount of ram on raspberry pi4 matter?

### DISCUSSION THREAD

**Discussion by akasal** [revoxhere/duino-coin#647] [Q&A] [answered]
Great looking project! I'm really excited about this.

Does the amount of ram improve hash? or is this all compute?

I'm going to buy some new boards for this and want to choose wisely. Thanks!

**Comment by dansinclair25** [comment, marked as answer]
It depends entirely on the algorithm you use. DUCO-S1 is CPU based, XXHASH is CPU and RAM.

*Please remember: more hash does not equal more reward*.

You’d be far better off buying some Arduino Uno/Nano boards (or even some ESP32 boards) and using a Pi as a low powered host.

**Reply by akasal** [reply]
Thanks for the info! I appreciate it.

one more question though... If duco-s1 is cpu based, then I'd still benefit from using some shiny new pi4's, right? As long as I'm starting from scratch, why not go for bigger hash? I'm doing it for fun, but part of the fun is to hit some heights, imo. Did you suggest uno's or nano's just for the lower cost? Thanks again for the feedback @dansinclair25

**Reply by akasal** [reply]
@dansinclair25 I think I found the answer by reading the white paper :) there are variable difficulty levels to normalize the differing hardware options. That's why more hash doesn't affect reward. Got it. Thanks again for your answer above.
